frontend/src/components/shared/AssessmentTreeNavigation.tsx

```
@@ -392,84 +379,46 @@ const AssessmentTreeNavigation: React.FC<AssessmentTreeNavigationProps> = ({
-
-        {/* SECTION 3: PREMIUM FEATURES */}
-        {projectId && !hidePremiumFeaturesButton && (
+    <div className="relative flex-shrink-0" style={{ width: sidebarWidth }}>
+      {/* Resize Handle - Positioned on the LEFT edge because layout is reversed */}
+      <div
+        className="absolute left-0 top-0 bottom-0 w-1 cursor-ew-resize hover:bg-primary/50 z-50 transition-colors"
+        onMouseDown={(e) => {
+          e.preventDefault();
+          setIsResizing(true);
+        }}
+      />
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)